assets/js/util.js

```diff
@@ -308,17 +332,21 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(useAjax) {
+        return $.ajax(requestOptions);
```

Comment:
> My question here is that a few lines below this one, you wrote code to switch to POST dynamically when needed. However, we bypass that protection here.

Yes. So the code to switch to POST is needed when we exceed a threshold request size. I thought we wouldn't encounter a case wherein the request size with URL will reach that size. Thus, I am keeping AJAX it with GET only. Doing this will help further where we would be able to wrap this around `$.jsonp()` which is what I was trying to say in previous comment. Do you suggest putting this `useAjax` flag along with the check condition something like:

```
if(requestUrl.length > APY_REQUEST_URL_THRESHOLD_LENGTH || useAjax) {} 
```